core/modules/mod.rs

```diff
@@ -253,12 +261,24 @@ impl ModuleSource {
       Some(found.into())
     };
     Self {
-      code: ModuleCode::from_static(code),
+      code: ModuleSourceCode::String(FastString::from_static(code)),
       module_type: ModuleType::JavaScript,
       module_url_specified: specified.into(),
       module_url_found: found,
     }
   }
+
+  pub fn get_string_source(
+    code: ModuleSourceCode,
+  ) -> Result<ModuleCode, FromUtf8Error> {
+    match code {
+      ModuleSourceCode::String(code) => Ok(code),
+      ModuleSourceCode::Bytes(bytes) => {
+        let str_ = String::from_utf8(bytes.to_vec())?;
```

Comment:
This performs a clone. Maybe I should store `Vec<u8>` instead in `ModuleSourceCode`?